assets/js/blocks/single-product/edit/edit-product-link.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, external } from '@woocommerce/icons';
+import { ADMIN_URL } from '@woocommerce/settings';
+import { Card, CardBody } from '@wordpress/components';
+
+/**
+ * Component to render an edit product link in the sidebar.
+ */
+const EditProductLink = ( { productId } ) => {
+	return (
+		<Card className="wc-block-single-product-edit-card">
```

Comment:
Should the class name be `wc-block-single-product__edit-card` instead? This also applies to the two class names below. :point_down:

Reply:
Also, I'm a bit confused. What are the benefits on using a `Card` over a normal `div`? The feedback prompt, which is a component quite similar, is using a `div`.

Reply:
I saw the Block Name + Description in the inspector uses cards so I followed suit. Should I not?

Reply:
You mean the `.block-editor-block-card`?

![imatge](https://user-images.githubusercontent.com/3616980/83245231-bc0c8d80-a1a0-11ea-8e74-ebbd8759fa8e.png)

Even though it's named `card`, I don't think it uses the `<Card>` component internally. At least, if I search for `.components-card` in the inspector I can only see one result (the one introduced in this PR).

Reply:
@Aljullu Thanks I see now. I just pushed the changes to use DIV.